sentry-samples/sentry-samples-android/src/main/java/io/sentry/sample/MainActivity.java

```diff
@@ -33,13 +33,28 @@ protected void onCreate(Bundle savedInstanceState) {
           Sentry.setExtra("extra", "extra");
           Sentry.setFingerprint(Collections.singletonList("fingerprint"));
           Sentry.setTransaction("transaction");
-          User user = new User();
-          user.setUsername("username");
-          Sentry.setUser(user);
-          Sentry.setTag("tag", "tag");
           Sentry.captureException(new Exception("Some exception with scope."));
         });
 
+    binding.unsetUser.setOnClickListener(
+      view -> {
+        Sentry.setTag("user_set", "null");
```

Comment:
```suggestion
        Sentry.removeTag("user_set");
```

Reply:
this would need to be implemented on the observer as well

Reply:
I wanted to have a tag to clearly see that the user was removed.
Maybe a better name?

Reply:
got it, like filtering out an existent tag, but IMO this should be using anyway the `remove` method, unless the `tag` still should exist with a null value, we can think about this at a later point.